### PR TITLE
"Open settings" button in the Camera permission rationale dialog

### LINKED ISSUE #422: Improve the permission rationale dialog UI
When the rationale dialog is shown there's no option to go to the App settings or continue with the permission flow. Let's see if this can be improved. 

Reported here - https://github.com/wordpress-mobile/WordPress-Android/pull/21312#issuecomment-2446964488

### PR DESCRIPTION
**Pull request by AdamGrzybkowski**
Closes #422 

### Description

The camera permission rationale dialog lacked the option to open the App settings directly. This PR changes that. 

Unfortunately, I couldn't find a way to launch the permission settings directly.

https://github.com/user-attachments/assets/f5ff3fea-6e90-4cc2-a12a-72ba99cc4dea

### Testing Steps

1. Go to /demo-app/AndroidManifest.xml and uncomment the camera permission 
2. Start the demo app
3. Try to take a picture
4. Decline the persission
5. Rationale Dialog should be shown - tap the `open settings`
6. Confirm the Demo App settings were opened

### CODE REVIEW

File: .idea/inspectionProfiles/Project_Default.xml [hunk 1]
```diff
@@ -1,44 +0,0 @@
-<component name="InspectionProjectProfileManager">
```
Comment (hamorillo) [inline .idea/inspectionProfiles/Project_Default.xml:1]: 🤔 I have some doubt about removing this file based on [this](https://intellij-support.jetbrains.com/hc/en-us/articles/206544839-How-to-manage-projects-under-Version-Control-Systems)

File: .idea/vcs.xml [hunk 2]
```diff
@@ -1,6 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (hamorillo) [inline .idea/vcs.xml:1]: 🤔 I have some doubt about removing this file based on [this](https://intellij-support.jetbrains.com/hc/en-us/articles/206544839-How-to-manage-projects-under-Version-Control-Systems) however taking a look at [gitignore.io](https://www.toptal.com/developers/gitignore/api/androidstudio,android) they would exclude it.

**Review by hamorillo — APPROVED** [review]
Looks good to me! I left some comments about the removed `./idea` files.

> I couldn't find a way to launch the permission settings directly

Same here. I couldn't find anything, so I think that works. I'll pay attention to other apps and see if any can navigate directly to the permissions screen.

🚀

File: .idea/inspectionProfiles/Project_Default.xml (hunk 1 above)
Comment (AdamGrzybkowski) [reply .idea/inspectionProfiles/Project_Default.xml:1]: I know that I didn't make any changes to our project config that should be shared, yet those files were modified. If this is some automated stuff by AS, I see no reason to share it. I haven't dug deep into it though...

**Pull request merged**